game/random_strategy.py

```diff
@@ -0,0 +1,9 @@
+from strategy import Strategy
+import random
+
+class RandomStrategy(Strategy):
+    def return_column(self, board, color):
+        while True:
```

Comment:
This potentially wastes a lot of cycles unnecessarily until you find a valid column. You would also loop if the board is full. You have `playable_positions` in `GameBoard` and a very simple way to pick a random value from there: http://stackoverflow.com/questions/306400/how-do-i-randomly-select-an-item-from-a-list-using-python